src/NCMenuBar.h

```diff
@@ -141,6 +132,16 @@ class NCMenuBar: public YMenuBar, public NCWidget
     NCMenuBar & operator=( const NCMenuBar & );
     NCMenuBar( const NCMenuBar & );
 
+    void select_next_menu();
```

Comment:
Please follow the conventions of the existing code. We don't use `snake_case()` in our C++ code, we use `camelCase()` for functions or `CamelCase()` for classes.

Yes, libyui-ncurses is a poor example for that because it has many authors and a long history. Still, we want to clean up stuff, not introduce new inconsistencies.